### DISCUSSION TITLE
Problem with Captcha in submitRateLimitChallenge

### DISCUSSION THREAD

**Discussion by DerRick79** [AsamK/signal-cli#1463] [Q&A]
Hey everybody.

I am new to signal-cli. I achieved registering my account (using a captcha) and also receiving a message. When I tried to send a first message, I got the following answer:
`
+491577xxxxxxx: CAPTCHA proof required for sending to "+491577xxxxxxx", available options "PUSH_CHALLENGE, RECAPTCHA" with challenge token "0d276e96-d774-4624-b1ea-ef85faxxxxxx", or wait "86400" seconds.
To get the captcha token, go to https://signalcaptchas.org/challenge/generate.html
After solving the captcha, right-click on the "Open Signal" link and copy the link.
Use the following command to submit the captcha token:
signal-cli submitRateLimitChallenge --challenge CHALLENGE_TOKEN --captcha CAPTCHA_TOKEN
1708359760xxx
`

So again I solved a token and submitted it with:
`signal-cli submitRateLimitChallenge --challenge 0d276e96-d774-4624-b1ea-ef85faxxxxxx --captcha signalcaptcha://signal-hcaptcha.5fad97ac-veryverylongtoken`

I got the response:
`Submit challenge error: Unable to parse entity`

Can anybody help me?

Thanks in advance,
Patrick

**Comment by cuu508** [comment]
Perhaps it's a shell escaping issue – try with double quotes around the token:

````
signal-cli submitRateLimitChallenge --challenge 0d276e96-d774-4624-b1ea-ef85faxxxxxx --captcha "signalcaptcha://signal-hcaptcha.5fad97ac-veryverylongtoken"
````
